src/containers/Bookings.js

```diff
@@ -2,30 +2,68 @@ import React, { Component } from 'react';
 import Search from '../components/Search.js';
 import ResultTable from '../components/Results.js';
 import FakeBookings from '../data/fakeBookings.json';
-import moment from 'moment';
+
+//import moment from 'moment';
 
 import '../results.css';
 
 export default class Bookings extends Component {
-  search = () => {
-    console.info('TO DO!');
-  };
-  render() {
-    //console.log(FakeBookings)
-
-    return (
-
-      <div className="App-content">
-        <div className="container">
-          <Search search={this.search}  />
-          <ResultTable data={FakeBookings} />
-          
-
-          {/* <FakeBookings  /> */}
-          
-          {/*<Results results={this.state.results} />*/} 
-        </div>
-      </div>
-    );
-  }
+
+    constructor() {
+        super()
+        this.state = {
+            filterName: " ",
+            FakeBookings: [],
+            filteredBookings: []
+        }
+    }
+
+    componentWillMount() {
+        this.setState({
+            FakeBookings,
+            filteredBookings: FakeBookings
```

Comment:
Interesting use of `componentWillMount`. We're going to cover this in detail in the class. In this case, personally I would initialise the state in the constructor instead. 

However, in the next homework, we will need to use `componentWillMount` like similar to this.